### PR TITLE
TELCODOCS-676: Update variables in RAN docs

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.11, 4.12

Issue: https://issues.redhat.com/browse/TELCODOCS-676

Link to docs preview:
https://49291--docspreview.netlify.app/openshift-enterprise/latest/monitoring/using-rfhe.html#nw-rfhe-installing-operator-cli_using-rfhe
https://49291--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-talm-for-cluster-upgrades.html#installing-topology-aware-lifecycle-manager-using-cli_cnf-topology-aware-lifecycle-manager
https://49291--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected.html#enabling-assisted-installer-service-on-bare-metal_ztp-deploying-disconnected
https://49291--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected.html#ztp-using-pgt-to-update-source-crs_ztp-deploying-disconnected
https://49291--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected.html#ztp-filtering-ai-crs-using-siteconfig_ztp-deploying-disconnected
https://49291--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected.html#ztp-preparing-the-ztp-git-repository_ztp-deploying-disconnected
https://49291--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected.html#ztp-adding-new-content-to-gitops-ztp_ztp-deploying-disconnected
https://49291--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected.html#ztp-deploying-a-site_ztp-deploying-disconnected
https://49291--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected.html#ztp-preparing-for-the-gitops-ztp-upgrade_ztp-deploying-disconnected
https://49291--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected.html#talo-platform-update_ztp-deploying-disconnected
https://49291--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected.html#talo-operator-update_ztp-deploying-disconnected

Additional information: `Branch Build` will be resolved as the version in the live docs

### CODE REVIEW

File: modules/ztp-roll-out-the-configuration-changes.adoc [hunk 1]
```diff
@@ -7,5 +7,5 @@
 = Roll out the configuration changes
 
 If any configuration changes were included in the upgrade due to implementing recommended changes, the upgrade process results in a set of policy CRs on the hub cluster in the `Non-Compliant` state. As of the {product-title} 4.10 release, these policies are set to `inform` mode and are not pushed to the spoke clusters without an additional step by the user. This ensures that potentially disruptive changes to the clusters can be managed in terms of when the changes are made, for example, during a maintenance window, and how many clusters are updated concurrently.
-
+// Please check line 9 for version. Does it need to be bumped to 4.11?
```
Comment (imiller0) [inline modules/ztp-roll-out-the-configuration-changes.adoc:10]: Similar to the above note the context is slightly misleading. The version is static here as well, indicating when the change in policy mode was made. Maybe this for wording:
```
As of the 4.10 version of the `ztp-site-generate` container, these policies are ...
```

File: modules/ztp-preparing-the-hub-cluster-for-ztp.adoc [hunk 2]
```diff
@@ -11,6 +11,7 @@ You can configure your hub cluster with a set of ArgoCD applications that genera
 .Prerequisites
 
 * Openshift Cluster 4.8 or 4.9 as the hub cluster
+// Please check line 13 for version. Do these need to be bumped to a later version?
 * {rh-rhacm-first} Operator 2.3 or 2.4 installed on the hub cluster
 * Red Hat OpenShift GitOps Operator 1.3 on the hub cluster
```
[the remark this answers is not in the document]
Comment (amolnar-rh) [reply modules/ztp-preparing-the-hub-cluster-for-ztp.adoc]: @imiller0 This PR is for 4.11, so please let me know what is needed for 4.11. 

If the versions are not correct in 4.10, please open a doc bug and send it to me.

[the remark this answers is not in the document]
Comment (yliu127) [reply modules/ztp-preparing-the-hub-cluster-for-ztp.adoc]: @amolnar-rh  suggest to recommend the following version for 4.11
hub: 4.11
ACM: 2.5  (we are seeing issues with ACM 2.6, we can probably add 2.6 after the blocking issue is fixed.)
GitOps: 1.5

[the remark this answers is not in the document]
Comment (amolnar-rh) [reply modules/ztp-preparing-the-hub-cluster-for-ztp.adoc]: @yliu127 Thanks for the help, I updated it accordingly

File: modules/cnf-topology-aware-lifecycle-manager-installation-cli.adoc [hunk 3]
```diff
@@ -51,10 +51,10 @@ $ oc get csv -n openshift-operators
 ----
 +
 .Example output
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-NAME                                                      DISPLAY                            VERSION               REPLACES   PHASE
-topology-aware-lifecycle-manager.4.10.0-202206301927      Topology Aware Lifecycle Manager   4.10.0-202206301927              Succeeded
+NAME                                                             DISPLAY                            VERSION                      REPLACES              PHASE
+topology-aware-lifecycle-manager.{product-version}.0-xxxxxxxxxxxx     Topology Aware Lifecycle Manager   {product-version}.0-xxxxxxxxxxxxrelease                        Succeeded
```
Comment (yliu127) [inline modules/cnf-topology-aware-lifecycle-manager-installation-cli.adoc:57]: For 4.10, the csv version format is 4.10.0-xxxxxxxxxxxx
but for 4.11+, the version is changed to 4.11.x  (e.g., 4.11.0, 4.11.1)

File: modules/ztp-manually-install-a-single-managed-cluster.adoc [hunk 4]
```diff
@@ -51,22 +51,20 @@ provisioner_cluster_registry }}/ocp4:{{ mirror_version_spoke_release }}
...
 . Create a `ClusterImageSet` for each specific cluster version that needs to be deployed. A `ClusterImageSet` has the following format:
 +
-[source,yaml]
+[source,yaml,subs="attributes+"]
 ----
 apiVersion: hive.openshift.io/v1
 kind: ClusterImageSet
 metadata:
-  name: openshift-4.9.0-rc.0 <1>
+  name: openshift-{product-version}.0-rc.0 <1>
```
Comment (yliu127) [inline modules/ztp-manually-install-a-single-managed-cluster.adoc:65]: please remove "-rc.0" from the name, so it matches with the releaseImage defined below.

File: modules/cnf-topology-aware-lifecycle-manager-installation-cli.adoc (hunk 3 above)
Comment (aireilly) [reply modules/cnf-topology-aware-lifecycle-manager-installation-cli.adoc]: fixed

File: modules/ztp-manually-install-a-single-managed-cluster.adoc (hunk 4 above)
Comment (aireilly) [reply modules/ztp-manually-install-a-single-managed-cluster.adoc]: fixed

**Pull request merged**